webpack.config.js

```diff
@@ -73,10 +73,10 @@ module.exports = (env, argv) => {
         remotes: {
           // TODO: Fetch this info from the shell DB.
           remote1: isProduction
-            ? "remote1@https://spotted-list.surge.sh/remoteEntry.js"
+            ? "remote1@http://apps.nov4.io/remote1/remoteEntry.js"
```

Comment:
Could each remote have their own subdomain? I assume that’s what we’ll do in prod, right?

Reply:
Yes we could 100% use a subdomain here.